src/backend/Shared/EventBus/EventBus.Extensions.Microsoft.DependencyInjection/Models/DefaultConnectionFactory.cs

```diff
@@ -0,0 +1,16 @@
+﻿using RabbitMQ.Client;
+
+namespace EventBus.Extensions.Microsoft.DependencyInjection.Models
+{
+    public class DefaultConnectionFactory : ConnectionFactory
+    {
+        public DefaultConnectionFactory(string hostName = "localhost")
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use the overloading mechanism instead of the optional parameters.](https://app.codacy.com/manual/iamprovidence/Lama/pullRequest?prid=4794247)